src/ur_driver.cpp

```diff
@@ -235,16 +239,19 @@ bool UrDriver::openServo() {
 	reverse_connected_ = true;
 	return true;
 }
-void UrDriver::closeServo(std::vector<double> positions) {
-	if (positions.size() != 6)
-		UrDriver::servoj(rt_interface_->robot_state_->getQActual(), 0);
-	else
-		UrDriver::servoj(positions, 0);
-
+void UrDriver::closeServo(const std::vector<double>& positions) {
+  assert (positions.size() == 6);
```

Comment:
IMHO a size different from 6 on a 6 DoF robot is a syntactix error, but I agree that we can just print a ROS_ERROR instead of asserting